src/analysis/calculus/fderiv.lean

```diff
@@ -2086,6 +2087,44 @@ lemma fderiv_const_mul (hc : differentiable_at 𝕜 c x) (d : 𝕜) :
 
 end mul
 
+section algebra_inverse
+variables {R :Type*} [normed_ring R] [normed_algebra 𝕜 R] [complete_space R]
+open normed_ring continuous_linear_map ring
+
+/-- At an invertible element `x` of a normed algebra `R`, the Fréchet derivative of the inversion
+operation is the linear map `λ t, - x⁻¹ * t * x⁻¹`. -/
+lemma has_fderiv_at_inverse  (x : units R) :
```

Comment:
Can you prove the stronger version `has_strict_fderiv_at` following the same lines?

Reply:
Maybe I tried this the wrong way, but it did not come out nicely.  (Did you try it yourself?)  I am planning to do smoothness in the next PR, via the bilinear map trick in Dieudonné -- what do you think about deducing the strict differentiability indirectly from smoothness, rather than proving it directly?

Reply:
No, I didn't try and it was a naive question: if it doesn't work out well, I say let's go for the usual differentiability, at least in this PR. 

(Note that higher smoothness only implies strict differentiability over reals or complexes, not in general.)